client/js/index.js

```diff
@@ -0,0 +1,6231 @@
+// Generated by purs bundle 0.12.0
```

Comment:
Should this file be ignored?

Reply:
I had just added a it to `.gitignore` right before opening the PR, but forgot to remove it. Fixed.